label_studio/tasks/api.py

```diff
@@ -192,7 +192,11 @@ def get_queryset(self):
             kwargs = {'all_fields': True}
         project = self.request.query_params.get('project') or self.request.data.get('project')
         if not project:
-            project = Task.objects.get(id=self.request.parser_context['kwargs'].get('pk')).project.id
+            try:
```

Comment:
I don't feel it is the right way to fix it - try simply returning 404 instead of 500

Reply:
Fixed, moved to get_object_or_404